webpack/assets/javascripts/react_app/components/SettingsTable/SettingsTable.js

```diff
@@ -1,18 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { Table } from '../common/table';
+import { Table, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
 
-import createSettingsTableSchema from './SettingsTableSchema';
+import { translate as __ } from '../../common/I18n';
+import SettingNameCell from './components/SettingName';
+import SettingValueCell from './components/SettingValueCell';
 
-const SettingsTable = ({ settings }) => (
-  <Table
-    key="settings-table"
-    columns={createSettingsTableSchema}
-    rows={settings}
-  />
+const columnNames = {
+  name: 'Name',
+  value: 'Value',
+  description: 'Description',
+};
+
+const TableHead = () => (
+  <Thead>
+    <Tr ouiaId="setting-table-heading-row">
+      <Th width={30}> {__(columnNames.name)} </Th>
+      <Th width={35}> {__(columnNames.value)} </Th>
+      <Th width={35}> {__(columnNames.description)} </Th>
```

Comment:
translation __ needs to get the string directly, and not through a variable, more info: https://projects.theforeman.org/projects/foreman/wiki/Translating